### ISSUE TITLE
how to detect a sentence is pure emoji or not?

### ISSUE THREAD

**Issue by jinfagang** [petehouston/flutter-emoji#10]
is there any API to do:

```
parser.is_pure_emoji('ddf')
```
if is full emoji, then return yes, otherwise return false.

**Comment by petehouston** [comment]
`'ddf'` is a text. I don't understand what you want. Can you elaborate?

**Comment by jinfagang** [comment]
Yes. I want detection is a text contains pure emoji or not. Such as:

😀 😁 😂 -> true
😀 😁 的还 -> not
😀 😁 fere😂 -> not
ferg😀 😁 😂 -> not

I want to do this is because I want detect if use's input is pure emoji or not, if pure emoji then I will enlarge all text say size as 99, so it will looks big just like facebook Messagers does, but if emoji with normal text, then just show normal.

Does it able to do this kind of detect?

**Comment by petehouston** [comment]
Yes, it is possible. But are you trying to capture text everytime user type to enlarge or only do once after user prompt the text?

**Comment by jinfagang** [comment]
@petehouston You are right, this should consider efficiency. I simply want detect it once user press enter button, then I will decide how to show it (if pure emoji then txt size big, other normal).

Will will do very fast to determine?
